src/ducks/selection/SelectionBar.jsx

```diff
@@ -0,0 +1,60 @@
+import React, { useContext, useCallback } from 'react'
+
+import UISelectionBar from 'cozy-ui/transpiled/react/SelectionBar'
+import Alerter from 'cozy-ui/transpiled/react/Alerter'
+import { useI18n } from 'cozy-ui/transpiled/react/I18n'
+
+import { SelectionContext } from 'ducks/context/SelectionContext'
+import useSelectionBarActions from 'ducks/selection/useSelectionBarActions'
+import { useTransactionCategoryModal } from 'ducks/transactions/TransactionRow'
+
+const SelectionBar = () => {
+  const {
+    isSelectionModeEnabled,
+    isSelectionModeActive,
+    selected,
+    emptySelection
+  } = useContext(SelectionContext)
```

Comment:
I prefer when `useContext(SelectionContext)` is moved in `SelectionContext' file.

Like that: 
```
export const useSelectionContext = () => {
  return useContext(SelectionContext)
}
```

It avoids importing useContext and the context, what to you think ?

Reply:
oui ok :)